pkgpanda/integration-tests/test_http.py

```diff
@@ -0,0 +1,153 @@
+import json
+import os
+from shutil import copytree
+
+from pkgpanda.http import app
+
+
+def _set_test_config(app):
+    app.config['TESTING'] = True
+    app.config['DCOS_ROOT'] = '../tests/resources/install'
+    app.config['DCOS_REPO_DIR'] = '../tests/resources/packages'
+
+
+def test_list_packages():
+    _set_test_config(app)
+    client = app.test_client()
+    packages = json.loads(client.get('/').data.decode('utf-8'))
+    assert packages == [
+        'mesos--0.22.0',
+        'mesos--0.23.0',
+        'mesos-config--ffddcfb53168d42f92e4771c6f8a8a9a818fd6b8',
+        'mesos-config--justmesos']
+
+
+def test_get_package():
+    _set_test_config(app)
+    client = app.test_client()
+
+    response = client.get('/mesos--0.22.0')
+    assert response.status_code == 200
+    assert json.loads(response.data.decode('utf-8')) == {
+        'id': 'mesos--0.22.0',
+        'name': 'mesos',
+        'version': '0.22.0',
+    }
+
+    response = client.get('/nonexistent-package--fakeversion')
```

Comment:
Can we test a invalid package name / version number? (`client.get('/!@#*')`, `client.get('/fooASDF')`